tests/acceptance/features/apiActivities/shareActivities.feature

```diff
@@ -1076,3 +1076,47 @@ Feature: check share activity
         }
       }
       """
+
+
+  Scenario: sharee tries to check the activities of a shared folder using share mount-point id
+    Given user "Alice" has created folder "/FOLDER"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | /FOLDER  |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
+    And user "Brian" has a share "/FOLDER" synced
+    And user "Brian" has uploaded file with content "some data" to "Shares/FOLDER/newfile.txt"
+    And user "Brian" has uploaded file with content "edited data" to "Shares/FOLDER/newfile.txt"
+    And user "Brian" has deleted file "Shares/FOLDER/newfile.txt"
+    When user "Brian" lists the activities of folder "FOLDER" from space "Shares" using the Graph API
+    Then the HTTP status code should be "403"
+
+  @issue-9860
+  Scenario: sharee tries to check the activities of unshared file
+    Given user "Alice" has uploaded file with content "another ownCloud test text file" to "anotherTextfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Brian" tries to list the activities of file "anotherTextfile.txt" owned by user "Alice" from space "Personal" using the Graph API
+    Then the HTTP status code should be "403"
+
+  @issue-9849
+  Scenario: sharee tries to check the activities of a shared folder using folder id
+    Given user "Alice" has created folder "/FOLDER"
+    And user "Alice" has uploaded file with content "some data" to "FOLDER/newfile.txt"
+    And user "ALice" has uploaded file with content "edited data" to "FOLDER/newfile.txt"
+    And user "ALice" has deleted file "FOLDER/newfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | /FOLDER  |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
+    And user "Brian" has a share "/FOLDER" synced
+    When user "Brian" lists the activities of folder "FOLDER" from space "Shares" using the Graph API
```

Comment:
These scenarios must not use the same When step:
```feature
Scenario: sharee tries to check the activities of a shared folder using share mount-point id
Scenario: sharee tries to check the activities of a shared folder using folder id
```

Reply:
also put them together.

Reply:
So, should we make a new step to list activities of shared file/folder using share mount-point id?

Reply:
yeah